theano/gpuarray/tests/test_basic_ops.py

```diff
@@ -182,6 +182,19 @@ def test_transfer_cpu_gpu():
     assert numpy.all(fv == av)
 
 
+def test_transfer_gpu_gpu():
+    g = GpuArrayType(dtype='float32', broadcastable=(False, False),
+                     context_name=test_ctx_name)()
+
+    av = numpy.asarray(rng.rand(5, 4), dtype='float32')
+    gv = gpuarray.array(av, context=get_context(test_ctx_name))
+
+    f = theano.function([g], GpuToGpu(test_ctx_name)(g))
+    theano.printing.debugprint(f)
```

Comment:
done

On Tue, Jun 7, 2016 at 10:51 AM, abergeron <notifications@github.com> wrote:

> In theano/gpuarray/tests/test_basic_ops.py
> <https://github.com/Theano/Theano/pull/4600#discussion_r66085821>:
>
> > @@ -182,6 +182,19 @@ def test_transfer_cpu_gpu():
> >      assert numpy.all(fv == av)
> >
> >
> > +def test_transfer_gpu_gpu():
> > +    g = GpuArrayType(dtype='float32', broadcastable=(False, False),
> > +                     context_name=test_ctx_name)()
> > +
> > +    av = numpy.asarray(rng.rand(5, 4), dtype='float32')
> > +    gv = gpuarray.array(av, context=get_context(test_ctx_name))
> > +
> > +    f = theano.function([g], GpuToGpu(test_ctx_name)(g))
> > +    theano.printing.debugprint(f)
>
> Please don't leave this in.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4600/files/c5b94c1306a700d4766fc8b74b76c75e4ef0f4cc#r66085821>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALC-y62Lnk58F7ecQIHSMlcqjri9adSks5qJYVsgaJpZM4IveVv>
> .
>